modules/microshift-configuring-ovn.adoc

```diff
@@ -8,6 +8,9 @@
 
 An OVN-Kubernetes config file can be written to `/etc/microshift/ovn.yaml`. {product-title} uses default OVN-Kubernetes configuration values if an OVN-Kubernetes config file is not customized.
 
+//Q: does the file populate with values if created empty, or is none of this true?
+//Q: what's a procedure for creating this file?
```

Comment:
There's a default file below. The user just has to create the file using any text editor they prefer. In the docs elsewhere we use the bash here document approach to make it easy to copy and paste, so that would be something like

```
sudo cat >/etc/microshift/ovn.yaml <<EOF
ovsInit:
  disableOVSInit: false
  gatewayInterface: ""
  externalGatewayInterface: ""
mtu: 1400
EOF
```